TodoList.Web/Models/EditViewModel.cs

```diff
@@ -7,5 +8,6 @@ public class EditViewModel
         public Guid Id { get; set; }
         public string Title { get; set; }
         public string Content { get; set; }
+        public string Tags { get; set; }
```

Comment:
You want to limit the length of each tag to 3 or 3 tags for each item? If its the latter, what would we achieve? It's not a problem to implement, but in my opinion 3 tags is too few.

Reply:
I would like to limit the tags because, in the future, I am planning to add a recommender based on the context of the Todo.

I can open a separate issue for this if you are not able to do so.

**_This PR will merge either way._**

Reply:
Oh, alright, no problem, I will add a limit on the number of tags along with other fixes addressing your comments.